src/loaddb/load_common.cpp

```diff
@@ -600,6 +600,12 @@ namespace cubload
 	    continue;
 	  }
 
+	if (class_is_ignored)
```

Comment:
I think handle_batch from load_common.cpp:644 also should not be called when class is ignored

Reply:
actually I think in this case batch_buffer will be just empty and batch handler will just exit if the buffer is empty